src/Raven.Client/Documents/Operations/Revisions/DeleteRevisionsOperation.cs

```diff
@@ -0,0 +1,277 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Raven.Client.Documents.Commands;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Extensions;
+using Raven.Client.Http;
+using Raven.Client.Json;
+using Sparrow.Json;
+
+namespace Raven.Client.Documents.Operations.Revisions
+{
+    public sealed class DeleteRevisionsOperation : IMaintenanceOperation<DeleteRevisionsOperation.Result>
+    {
+        private readonly Parameters _parameters;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="DeleteRevisionsOperation"/> class.
+        /// This operation allows deletion of revisions.
+        /// </summary>
+        /// <remarks>
+        /// This is an <c>Admin</c> to operation.
+        /// </remarks>
+
+        /// <param name="documentIds">
+        /// A list of document IDs whose revisions you want to delete. 
+        /// Default is an empty list if not provided.
+        /// </param>
+        /// <exception cref="ArgumentNullException">
+        /// Thrown when the <paramref name="documentIds"/> parameter is null or empty.
+        /// </exception>
+
+        /// <param name="removeForceCreatedRevisions">
+        /// Indicates whether to include force-created revisions in the deletion. Default is <c>false</c>
```

Comment:
link to the docs or reference to the API would be nice I think.

Reply:
Done